config/7.8.0/pim/dbs/postgresql.yaml

```diff
@@ -0,0 +1,154 @@
+## KIE ProcessMigration BEGIN
+processMigration:
+  ## KIE ProcessMigration deployment config BEGIN
+  deploymentConfigs:
```

Comment:
I wonder whether we can somehow reuse the existing configurations for kieservers. That introduces a lot of duplicated configuration.

Reply:
@ruromero I have moved the deployment of mysql and postgresql to separate templates and reused by both kie server and process migration. Please take a look. Thanks.